packages/js/core/src/types/Client.ts

```diff
@@ -16,8 +14,7 @@ import { WrapManifest } from "@polywrap/wrap-manifest-types-js";
 import { Result } from "@polywrap/result";
 
 export interface ClientConfig<TUri extends Uri | string = string> {
-  readonly redirects: Readonly<UriRedirect<TUri>[]>;
-  readonly plugins: Readonly<PluginRegistration<TUri>[]>;
+  readonly redirects: Readonly<IUriRedirect<TUri>[]>;
```

Comment:
Do we still need to keep redirects in config if it can just be a resolver?

Reply:
We don't, but this PR is already quite large, so I'm focusing it on plugins and going to refactor redirects as a separate one